src/core/dbus-cgroup.c

```diff
@@ -346,6 +346,51 @@ static int property_get_ip_address_access(
         return sd_bus_message_close_container(reply);
 }
 
+static int property_get_restrict_network_interfaces(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+        int r;
+        CGroupContext *c = userdata;
+        _cleanup_strv_free_ char **l = NULL;
+
+#if BPF_FRAMEWORK
+        char *iface;
+#endif
+
+        assert(bus);
+        assert(reply);
+        assert(c);
+
+        r = sd_bus_message_open_container(reply, 'r', "bas");
+        if (r < 0)
+                return r;
+
+        r = sd_bus_message_append(reply, "b", c->restrict_network_interfaces_is_allow_list);
+        if (r < 0)
+                return r;
+
+#if BPF_FRAMEWORK
+        SET_FOREACH(iface, c->restrict_network_interfaces) {
+                r = strv_extend(&l, iface);
+                if (r < 0)
+                        return r;
+        }
+
+        strv_sort(l);
+#endif
+
+        r = sd_bus_message_append_strv(reply, l);
```

Comment:
instead of using sd_bus_message_append_strv() just use sd_bus_message_open_container() before the loop, sd_bus_message_append() in the loop, and sd_bus_message_close_container() after the loop. There's a lot of prior art in the codebase to see how that works. i.e. instead of first assemling an strv and then adding that to the messgae, just add things to the message right-away, do not assemble a strv